src/worker.js

```diff
@@ -457,6 +445,22 @@ const aioli = {
 		}
 	},
 
+	// =========================================================================
+	// Reinitialize a tool
+	// =========================================================================
+	async _reinit(tool) {
```

Comment:
Since this function is meant to be exposed, let's call it `reinit` without an underscore. Can you also move this block to line 273, just to keep the exposed functions co-located in the code?